src/components/pages/favorites/Favorites.tsx

```diff
@@ -0,0 +1,140 @@
+export default function Favorites() {
```

Comment:
```suggestion
export default function Favorites(): JSX.Element {
```

Давайте придерживаться явного объявления типов